docs/iris/src/whatsnew/aggregate_directory.py

```diff
@@ -238,7 +238,10 @@ def compile_directory(directory, release):
         for file_description in category_items:
             entry_path = os.path.join(directory, file_description['FileName'])
             with open(entry_path, 'r') as content_object:
-                category_text.extend(content_object.readlines())
+                text = content_object.readlines()
+                if not text[-1].endswith('\n'):
+                    text[-1] += '\n'
+                category_text.extend(text)
```

Comment:
Automatically fix troublesome contributions with no trailing new line ... otherwise multiple contribution entries are rendered on the same line.